packages/cra-external-template/template.json

```diff
@@ -0,0 +1,46 @@
+{
+  "package": {
+    "dependencies": {
+      "@reapit/cognito-auth": "^2.1.7",
```

Comment:
We can remove Cognito Auth now right?

Reply:
We will need type`LoginMode` exported by `@reapit/cognito-auth` in the menu component

Reply:
Should be refactored to remove LoginType - it is no longer required